src/connectors/search-box/connectSearchBox.js

```diff
@@ -158,8 +158,8 @@ export default function connectSearchBox(renderFn, unmountFn = noop) {
         };
       },
 
-      getWidgetSearchParameters(searchParameters, { uiState }) {
-        return searchParameters.setQueryParameter('query', uiState.query);
+      getWidgetSearchParameters(state, { uiState }) {
+        return state.setQueryParameter('query', uiState.query || '');
```

Comment:
Do you just want this to be consistent with the getConfiguration here? Not sure what flow will work best for all widgets.

Reply:
WDYM?

Reply:
Cleared it up in person, I was talking about the goal of the PR here, where we make sure the changes done in getConfiguraiton are replicated in getWidgetSearchParameters